contribute/bugreporting.rst

```diff
@@ -11,7 +11,7 @@ This might seem obvious, but it's easy to miss. Go to (Settings - Updates) and m
 Check if the bug is already reported
 ------------------------------------
 
-Open up the bug tracker for `ubuntu-touch <https://github.com/ubports/ubuntu-touch>`_.
+Open up the bug tracker for `ubuntu-touch <https://gitlab.com/ubports/ubuntu-touch>`_.
```

Comment:
Let's use the link to the issue tracker: https://gitlab.com/ubports/ubuntu-touch/-/issues

Reply:
The advantage of the other link is it shows the Readme and instructions there

Reply:
True. Then let's keep it as proposed in this PR.